server/remote_cache/action_cache_server/action_cache_server.go

```diff
@@ -242,3 +244,59 @@ func (s *ActionCacheServer) UpdateActionResult(ctx context.Context, req *repb.Up
 	}
 	return req.ActionResult, nil
 }
+
+// Inlines the contents of output files requested to be inlined as long as the
+// total size of the ActionResult is below maxResultSize.
+func (s *ActionCacheServer) maybeInlineOutputFiles(ctx context.Context, req *repb.GetActionResultRequest, ar *repb.ActionResult, maxResultSize int) {
```

Comment:
I could use some hints on how to write tests for `action_cache_server`.

Reply:
Looks like `action_cache_server_test` doesn't exist yet - it would be awesome if you could add that. You could use `content_addressable_storage_server_test` as an example.